packages/teams-js/src/public/appWindow.ts

```diff
@@ -46,6 +46,7 @@ export class ChildAppWindow implements IAppWindow {
    * @param listener - The listener that will be called
    */
   public addEventListener(type: string, listener: (message: any) => void): void {
+    ensureInitialized();
```

Comment:
Does it make sense to ensure initialization for only `FrameContexts.content`, `FrameContexts.sidePanel`,  and `FrameContexts.meetingStage`

Reply:
Since this is for back-compat only, I am fine with just leaving this as-is